doc/whats_new/v0.21.rst

```diff
@@ -89,6 +89,14 @@ Support for Python 3.4 and below has been officially dropped.
 :mod:`sklearn.decomposition`
 ............................
 
+- |Enhancement| :class:`decomposition.KernelPCA` now properly checks the
+  eigenvalues found by the solver for numerical or conditioning issues, relying
+  on new common util `utils.validation.check_psd_eigenvalues`. This ensures
+  consistency of results across solvers (different choices for `eigen_solver`),
```

Comment:
Should be double backticks here, same for ``` `'randomized'` and `'lobpcg'` ```

Reply:
ok. I also fixed it for the previous PR text (it seems that it was forgotten on the way)